src/main/java/at/hannibal2/skyhanni/config/features/rift/RiftConfig.java

```diff
@@ -55,4 +55,10 @@ public class RiftConfig {
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean horsezookaHider = false;
+
+    @Expose
+    @ConfigOption(name = "Ubik's cube reminder", desc = "Reminder for Ubik's cube in the rift")
```

Comment:
Please move it to the mountaintop category (i assume this is in mountaintop), and please describe WHAT it is you do with ubik a bit in the description. Also, the feature name is a title, please use first letter upper case formatting there, and a dot at the end of the description